# 
# I think this test is obsolete and can be deleted. However I'd like
# to check my suspicion on some other OS than Solaris.
# I could not find a module for rfc2228 in proftpd sources.
# there are just some traces in project history.
# However I have not tried to build it somewhere else yet
# to be absolutely sure. Keeping patch private to Solaris now.
#

--- a/tests/t/config/multilinerfc2228.t
+++ /dev/null
@@ -1,11 +0,0 @@
-#!/usr/bin/env perl
-
-use lib qw(t/lib);
-use strict;
-
-use Test::Unit::HarnessUnit;
-
-$| = 1;
-
-my $r = Test::Unit::HarnessUnit->new();
-$r->start("ProFTPD::Tests::Config::MultilineRFC2228");
